docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3'
+services:
+  db:
+    image: postgres
```

Comment:
You probably want to specify a version of the postgres image since data format are not necessary compatible between version, so if you redeploy and a new version has been release it may not boot because the format have change, and we need to migrate the data first.